src/lib/components/emoji/emoji.svelte

```diff
@@ -4,13 +4,14 @@
   import { CUSTOM_EMOJI_COMPONENTS } from './emoji';
 
   export let emoji: string;
-  export let size: 'small' | 'regular' | 'large' | 'huge';
+  export let size: 'small' | 'regular' | 'large' | 'huge' | 'massive';
```

Comment:
ChatGPT is feeling creative:


'minuscule' | 'modest' | 'substantial' | 'immense' | 'enormous'
'peewee' | 'ordinary' | 'grand' | 'titanic' | 'monstrous'
'micro' | 'midi' | 'maxi' | 'jumbo' | 'gargantuan'
'petite' | 'average' | 'ample' | 'humongous' | 'mammoth'
'pint-sized' | 'standard' | 'spacious' | 'stupendous' | 'ginormous'

😁

Reply:
lolllll "pint-sized"